cfme/tests/services/test_catalog_item.py

```diff
@@ -37,24 +41,42 @@ def dialog():
     flash.assert_success_message('Dialog "{}" was added'.format(dialog))
     yield dialog
 
+    try:
+        service_dialog.delete()
+    except NoSuchElementException:
+        logger.info('test_catalog_item: dialog yield fixture cleanup, dialog not found')
```

Comment:
could we override logging's pattern or change default one to show test's file/name by default ?